### PR TITLE
Clarify a little bit the Fault Injection task

### PR DESCRIPTION
**Pull request by jwendell**
By stating what's the initial state and fixing a wrong result
in the abort test.

### CODE REVIEW

File: content/docs/tasks/traffic-management/fault-injection/index.md [hunk 1]
```diff
@@ -188,8 +192,10 @@ ratings not available` message.
     If the rule propagated successfully to all pods, the page loads
     immediately and the `Ratings service is currently unavailable` message appears.
 
-1. Log out from user `jason` and the rating stars show up successfully on the
-application's `/productpage`.
+1. If you log out from user `jason` or open the Bookinfo application in an anonymous
+   window (or in another browser), you will see that `/productpage` still calls `reviews:v1`
+   (which does not call `ratings` at all) for everybody but `jason`, therefore you
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/fault-injection/index.md]: s/, therefore/. Therefore/

File: content/docs/tasks/traffic-management/fault-injection/index.md [hunk 2]
```diff
@@ -188,8 +192,10 @@ ratings not available` message.
...
+   shall not see any error message.
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/fault-injection/index.md]: will not see any error messages.

File: content/docs/tasks/traffic-management/fault-injection/index.md [hunk 3]
```diff
@@ -28,6 +28,10 @@ This task shows you how to inject faults to test the resiliency of your applicat
     $ kubectl apply -f @samples/bookinfo/networking/virtual-service-reviews-test-v2.yaml@
     {{< /text >}}
 
+* This is how the requests look like:
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/fault-injection/index.md]: With the above configuration, this is how requests flow:

**Comment by jwendell** [comment]
@frankbu thanks for the review. Updated according.

**Pull request merged**